tests/disaster-recovery/regional-dr/test_node_failure.py

```diff
@@ -0,0 +1,184 @@
+            pytest.param(
+                "submariner-gateway", marks=pytest.mark.polarion_id("OCS-4439")
+            ),
+        ],
+    )
+    def test_managed_cluster_node_failure(
+        self, resource_on_node, nodes_multicluster, rdr_workload, node_restart_teardown
+    ):
+        """
+        Tests to fail managed cluster's node where the given resource is hosted,
+        verify the pod is rescheduled on a healthy node and mirroring is working as expected.
+        Node failures are initially induced separately in each managed cluster,
+        then simultaneously in both managed clusters.
+
+        """
+        managed_clusters = get_non_acm_cluster_config()
+
+        logger.info("Inducing node failure in each managed cluster separately")
+        for cluster in managed_clusters:
+            index = cluster.MULTICLUSTER["multicluster_index"]
+            config.switch_ctx(index)
+            logger.info(
+                f"Inducing node failure in cluster: {cluster.ENV_DATA['cluster_name']}"
+            )
+            resource = Pod(
+                **get_pods_having_label(
+                    label=self.label_ns_map[resource_on_node]["label"],
+                    namespace=self.label_ns_map[resource_on_node]["namespace"],
+                )[0]
+            )
+            resource_node = get_pod_node(resource)
+            logger.info(
+                f"Stopping {resource_node.name} where {resource.name} is hosted"
+            )
+            nodes_multicluster[index].stop_nodes([resource_node])
+            wait_for_nodes_status(
+                node_names=[resource_node.name], status=constants.NODE_NOT_READY
+            )
+
+            # Wait for pod to be rescheduled. For submariner-gateway, use sleep as it doesn't get rescheduled
+            if resource_on_node == "submariner-gateway":
+                logger.info("Waiting for 300 seconds before starting the node")
+                time.sleep(300)
+            else:
+                # Wait for pod to reach terminating state or to be deleted
+                assert wait_for_pods_to_be_in_statuses(
+                    [constants.STATUS_TERMINATING],
+                    pod_names=[resource.name],
+                    namespace=self.label_ns_map[resource_on_node]["namespace"],
+                    timeout=420,
+                    sleep=30,
+                )
+                # Wait for the new pod to reach running state
+                resource.ocp.wait_for_resource(
+                    condition=constants.STATUS_RUNNING,
+                    selector=self.label_ns_map[resource_on_node]["label"],
+                    resource_count=1,
+                )
+            nodes_multicluster[index].start_nodes([resource_node])
+            wait_for_nodes_status([resource_node.name])
+            logger.info(
+                "Wait for all the pods in openshift-storage to be in running state"
+            )
+            assert wait_for_pods_to_be_running(
```

Comment:
Do we run IO and validate consistency with replication intervals? if not, pls consider adding them in a subsequent PR